locales/usePageLocale.ts

```diff
@@ -9,10 +9,17 @@ const usePageLocale = <NS extends keyof PageLocalesResources>(_ns: NS) => {
   type Result = PageLocalesResources[NS]
 
   const { lang, values } = useLocaleContext()
-  const localeValues = values as Result
+  const localeValues = (values || {}) as Result
 
   function translate<NSK extends keyof Result>(key: NSK): Result[NSK] {
-    return localeValues[key] || (`[${key}]` as unknown as Result[NSK])
```

Comment:
The bug is here because empty string is `falsy`. I've checked and the key reported, `infoText`, is an `empty` string. 

The other error is about re-rendering in a different way that maybe the site is used to (client-side) so I've added an if statement to render the loading version of the site while we don't have this data.